dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -32,39 +32,13 @@ data:
       "links": [],
       "liveNow": false,
       "panels": [
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${datasource}"
-          },
-          "description": "",
-          "gridPos": {
-            "h": 3,
-            "w": 20,
-            "x": 0,
-            "y": 0
-          },
-          "id": 31,
-          "options": {
-            "code": {
-              "language": "plaintext",
-              "showLineNumbers": false,
-              "showMiniMap": false
-            },
-            "content": "This dashboard shows all SLOs for RHTAP over a specified time window.\n\nSelect a datasource to switch between environments.",
-            "mode": "markdown"
-          },
-          "pluginVersion": "9.3.8",
-          "transparent": true,
-          "type": "text"
-        },
```

Comment:
Please leave this panel of instructions at the top.

Reply:
Done and verified on test instance:

<img width="384" alt="image" src="https://github.com/redhat-appstudio/o11y/assets/84398375/9e6f9a56-be8e-45f3-b87c-9d70ed1a3089">